dir.props

```diff
@@ -197,6 +186,8 @@
     </When>
   </Choose>
 
+  <Import Project="$(BinDir)tools/configuration/configuration.props" Condition="Exists('$(BinDir)tools/configuration/configuration.props')" /> 
```

Comment:
Doesn't this need to be above the Choose statement looking at ConfigurationGroup? Otherwise that will never be set.